lib/schema/facter.yaml

```diff
@@ -369,6 +369,14 @@ filesystems:
         Mac OSX: The usable file systems is limited to the file system of mounted devices.
     blockgroup: file system
 
+in_fips_mode:
```

Comment:
Is it always boolean or can there be a "learning" mode like SELinux?

If boolean, what do you think about naming it `fips_enabled`?

Reply:
From what I know it's always boolean. We could name it as but it might have different connotations compared to actually being in fips mode. But no strong opinions.